.github/workflows/peernode.yml

```diff
@@ -38,11 +38,25 @@ jobs:
     - name: Test
       working-directory: peernode
       run: cargo test --verbose
+
+#    - name: Start server and run sea pig tests
+#      working-directory: peernode
+#      run: |
+#        cargo run --bin peer-node-server -F test_local_market &
+#        sleep 5 &&
+#        cargo test seapig_test --verbose -- --test-threads=1 --ignored
+#    - name: Start server and run bubble guppies tests
+#      working-directory: peernode
+#      run: |
+#        cargo build --bin peer-node-server -F test_local_market &&
+#        cargo run --bin peer-node-server -F test_local_market &
+#        sleep 10 &&
+#        cargo test bubbleguppies_test --verbose -- --test-threads=1 --ignored
+
```

Comment:
What does this do?

Reply:
I was attempting to run the tests on Github Actions by starting a server and then running a test but shelved it for now. The tests work locally if you start up a server and then run tests, not sure what's going on with github.

Reply:
Ok, so just add a TODO comment for now here